script/FullDeployer.s.sol

```diff
@@ -38,33 +45,53 @@ contract FullDeployer is HubDeployer, ExtendedSpokeDeployer {
         uint16 centrifugeId;
         string memory environment;
         string memory network;
+        address rootAddress;
 
         try vm.envString("NETWORK") returns (string memory _network) {
             network = _network;
             string memory configFile = string.concat("env/", network, ".json");
             string memory config = vm.readFile(configFile);
             centrifugeId = uint16(vm.parseJsonUint(config, "$.network.centrifugeId"));
             environment = vm.parseJsonString(config, "$.network.environment");
+
+            try vm.parseJsonAddress(config, "$.network.root") returns (address _root) {
+                rootAddress = _root;
+            } catch {
+                rootAddress = address(0);
+            }
+
+            // Parse batchGasLimit with defaults
+            try vm.parseJsonUint(config, "$.network.batchGasLimit") returns (uint256 _batchGasLimit) {
+                batchGasLimit = _batchGasLimit;
+            } catch {
+                batchGasLimit = 25_000_000; // 25M gas
```

Comment:
I think we don't need a default value here, unless you discussed something different with Jeroen